src/app/containers/Ad/index.jsx

```diff
@@ -1,53 +1,25 @@
 import React, { useContext } from 'react';
-import styled from 'styled-components';
 import pathOr from 'ramda/src/pathOr';
-import {
-  GEL_GROUP_3_SCREEN_WIDTH_MIN,
-  GEL_GROUP_4_SCREEN_WIDTH_MIN,
-} from '@bbc/gel-foundations/breakpoints';
-import {
-  GEL_SPACING_TRPL,
-  GEL_SPACING_QUAD,
-} from '@bbc/gel-foundations/spacings';
 import { RequestContext } from '../../contexts/RequestContext';
 import { ServiceContext } from '../../contexts/ServiceContext';
-
 import Amp from './Amp';
 import useToggle from '#hooks/useToggle';
 
-const StyledAd = styled.div`
-  /* To centre page layout for Group 4+ */
-  margin: 0 auto;
-  width: 100%; /* Needed for IE11 */
-  text-align: center;
-
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    margin-top: ${GEL_SPACING_TRPL};
-  }
-
-  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
-    margin-top: ${GEL_SPACING_QUAD};
-    max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN};
-  }
-`;
-
 const AdContainer = () => {
   const { isAmp } = useContext(RequestContext);
   const { service, ads } = useContext(ServiceContext);
   const hasAds = pathOr(false, ['hasAds'], ads);
   const { enabled: adsEnabled } = useToggle('ads');
 
+  // AMP does not run the useEffect, so it's defaulting to the hard coded toggle
+  // Currently only enabled for pidgin
```

Comment:
Do we need this comment? I've just ensured that our toggles service returns values for `ads` for all WS services on the test environment. 
```suggestion
```

Reply:
I've noticed that the local `hasAds` toggle is only enabled for Pidgin. Can we please disable that and enable it for these services instead:
afrique
arabic
hindi
mundo
portuguese
russian
zhongwen

These are front pages that should have advertising.

Reply:
Also, you'll need to update integration test snapshots after doing that. `npm run test:integration:ci -- -u`

Reply:
Thanks 👍